namerd/main/src/main/scala/io/buoyant/namerd/DtabHandler.scala

```diff
@@ -31,60 +31,40 @@ class DtabHandler(
 
   def renderDtab(name: String, dtab: Dtab): Future[Response] = {
     val body = s"""
-          <div class="container-fluid">
             <div class="row">
               <div class="col-lg-6">
                 <h2 class="router-label-title">Namespace "$name"</h2>
               </div>
             </div>
             <div class="delegator">
             </div>
-          </div>
 
           <script id="data" type="application/json">{"namespace": "$name", "dtab": ${
       DelegateApiHandler
         .Codec.writeStr(dtab)
     }}</script>
-          <script data-main="../files/js/main-namerd" src="../files/js/lib/require.js"></script>
+          <script data-main="/files/js/main-namerd" src="/files/js/lib/require.js"></script>
       """
     render(body)
   }
 
   def renderError(e: Throwable): Future[Response] = {
     val body = s"""
-          <div class="container-fluid">
             <div class="row">
               <div class="col-lg-6">
                 <h2 class="router-label-title">Dtab Error</h2>
                 <p>An error was encountered when fetching or parsing the dtab:</p>
                 <pre>$e</pre>
               </div>
             </div>
-          </div>
     """
     render(body)
   }
 
   def render(body: String): Future[Response] = {
-    val content = s"""
-      <!doctype html>
-      <html>
-        <head>
-          <title>namerd admin</title>
-          <link rel="shortcut icon" href="../files/images/favicon.png" />
-          <link type="text/css" href="../files/css/lib/bootstrap.min.css" rel="stylesheet"/>
-          <link type="text/css" href="../files/css/fonts.css" rel="stylesheet"/>
-          <link type="text/css" href="../files/css/dashboard.css" rel="stylesheet"/>
-          <link type="text/css" href="../files/css/delegator.css" rel="stylesheet"/>
-        </head>
-        <body>
-        $body
-        </body>
-      </html>
-    """
     val response = Response()
-    response.contentType = MediaType.Html + ";charset=UTF-8"
```

Comment:
Why is this being removed?

Reply:
This is removed so that [this](https://github.com/linkerd/linkerd/pull/2258/files#diff-e1ab574e9f83b4a3327606f6d43bbfa9R132) line in NamerdFilter can evaluate to true. The `contains` checks for the exact match of a string within an `Option`. So I had to change it to satisfy the condition so that we could build the final rendered HTML correctly.